packages/alfa-style/test/property/border-[block,inline]-[end,start]-color.spec.tsx

```diff
@@ -0,0 +1,69 @@
+import { test } from "@siteimprove/alfa-test";
+import { h } from "@siteimprove/alfa-dom/h";
+
+import { Device } from "@siteimprove/alfa-device";
+
+import { Style } from "../../src/style";
+
+const device = Device.standard();
+
+for (const box of ["block", "inline"] as const) {
```

Comment:
If we have all these `extend` the same base case, we can just test the base case (and maybe test `extend` separately).
This feels like complex test machinery just for testing 4 times the same code. If the test machinery is more complex than the code it tests, we're not doing it right.

Reply:
> This feels like complex test machinery just for testing 4 times the same code.

Yes and no; it started out testing different, but equal, code paths and now happen to be testing the same code path after an internal refactor. It still serves as a valuable proof that `Property.extends()` behaves as expected for these properties and will continue to serve as proof that all properties are correctly implemented even after future refactors.